src/audits/performance/speed-index-metric.js

```diff
@@ -19,9 +19,15 @@
 
 const speedline = require('speedline');
 const Audit = require('../audit');
+const TracingProcessor = require('../../lib/traces/tracing-processor');
 
 const FAILURE_MESSAGE = 'Navigation and first paint timings not found.';
 
+// Parameters (in ms) for log-normal CDF scoring. To see the curve:
+// https://www.desmos.com/calculator/mdgjzchijg
+const SCORING_FALLOFF = 1250;
```

Comment:
first, thanks for adding a full description of the scoring over in traceingprocessor.

can we call this scoring-falloff the point-of-dim-returns? I know it's long an unweildy, but it's very clear and reduces the amount of vocabulary we introduce to communicate how this works.

so i guess `const SCORING_POINT_OF_DIMINISHING_RETURNS = 1250`   :)